llarp/quic/connection.cpp

```diff
@@ -897,35 +897,42 @@ namespace llarp::quic
       fprintf(stderr, "add_stream_data for non-stream returned [%ld,%ld]\n", nwrite, consumed);
       assert(consumed <= 0);
 
-      switch (nwrite)
+      if (nwrite == -240)
```

Comment:
there isn't really a good reason to use switch case for anything not an `enum class`. the compiler is smart enough to optimize it.